Packs/Stairwell/Integrations/Inception/Inception.py

```diff
@@ -40,8 +40,10 @@ def file_enrichment_command(client, file_hash):
             responseJson = json.loads(response)
             md = '# Stairwell Inception\n'
             file_md5 = responseJson['data']['attributes']['md5']
```

Comment:
If 'md5' is not a key in attributes you will get a keyError exception. It is better to use responseJson['data']['attributes'].get('md')

Same for all the rest.

Reply:
In our API, it will always respond with MD5, SHA1, and SHA256. These are not optional fields in the reponse.